xblockutils/__init__.py

```diff
@@ -2,4 +2,4 @@
 Useful classes and functionality for building and testing XBlocks
 """
 
-__version__ = '3.0.0'
+__version__ = '3.0.1'
```

Comment:
This version breaks Maple compatibility because `MakoService` was introduced in Nutmeg. While Maple is an old release, it's good to bump the minor version and add a `fix!` to announce this. I did a similar thing in https://github.com/openedx/xblock-drag-and-drop-v2/pull/318.

Reply:
Sure, doing that now. Thanks!